pkg/interface/src/views/components/Author.tsx

```diff
@@ -1,28 +1,36 @@
 import React, {ReactNode} from "react";
 import moment from "moment";
+import { Row, Box } from "@tlon/indigo-react";
+
 import { Sigil } from "~/logic/lib/sigil"
 import { uxToHex, cite } from "~/logic/lib/util";
-import { Contacts } from "~/types/contact-update";
-import { Row, Box } from "@tlon/indigo-react";
+import { Contacts, Rolodex } from "~/types/contact-update";
+import OverlaySigil from "./OverlaySigil";
+import { Group, Association, LocalUpdateRemoteContentPolicy } from "~/types";
+import GlobalApi from "~/logic/api/global";
+import { useHistory } from "react-router-dom";
 
 interface AuthorProps {
-  contacts: Contacts;
+  contacts: Rolodex;
   ship: string;
   date: number;
   showImage?: boolean;
   hideAvatars: boolean;
   hideNicknames: boolean;
   children?: ReactNode;
+  remoteContentPolicy: LocalUpdateRemoteContentPolicy;
+  group: Group;
+  api: GlobalApi;
 }
 
-export function Author(props: AuthorProps) {
-  const { contacts, ship = '', date, showImage } = props;
-  let contact = null;
+export default function Author(props: AuthorProps) {
+  const { contacts, ship = '', date, showImage, hideAvatars, hideNicknames, remoteContentPolicy, group, api } = props;
+  const history = useHistory();
+  let contact;
   if (contacts) {
     contact = ship in contacts ? contacts[ship] : null;
```

Comment:
`contacts` here is assuming we have JUST the one group's contacts; but it's being passed the *entire* contacts prop. In LinkItem you will want to process the prop accordingly before passing it down. Note how this should say "galen" but doesn't — 

<img width="554" alt="Screen Shot 2020-11-24 at 8 16 55 PM" src="https://user-images.githubusercontent.com/20846414/100169794-3f7a1380-2e92-11eb-88fd-94655bbae9f7.png">